misc/extension_api_validation/4.3-stable.expected

```diff
@@ -122,3 +122,93 @@ GH-98918
 Validate extension JSON: Error: Field 'classes/FileAccess/methods/open_encrypted/arguments': size changed value in new API, from 3 to 4.
 
 Optional argument added to allow setting initialization vector. Compatibility method registered.
+
+
+GH-98972
+--------
+Validate extension JSON: Error: Field 'classes/TranslationServer/methods/standardize_locale/arguments': size changed value in new API, from 1 to 2.
+
+Optional argument added. Compatibility method registered.
```

Comment:
You've added a lot here that I'm not sure where it comes from, did you copy these over from the 4.2 file?

Reply:
I copied the lines from `4.3-stable.expected` and put my changes at the end. I thought that it would fix the message "This branch has conflicts that must be resolved". I did that because my changes got outdated from the new lines inside of the `4.3-stable.expected` file.

Reply:
You need to rebase your branch as you are behind and you are making these changes on top of older code, you shouldn't copy what's in these files right now but rebase instead and just add your content

Reply:
How can I do that exactly? The last time I tried it, I ended up accidentally ruining the entire PR. Sorry about the questions, I'm nervous about it that.

Reply:
See [the interactive rebase](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#the-interactive-rebase) for details

Reply:
Oh, so I need to update my branch with a rebase?
I have another question, should I squash my commits after I'm done?

Reply:
Yes you need to rebase your branch on the up-to-date branch, squashing would be good too but not necessary until this is approved